runtime/dart_isolate.cc

```diff
@@ -709,6 +709,34 @@ static void InvokeDartPluginRegistrantIfAvailable(Dart_Handle library_handle) {
   tonic::LogIfError(tonic::DartInvokeField(plugin_registrant, "register", {}));
 }
 
+namespace {
+bool EndsWith(const std::string& str, const std::string& ending) {
+  if (str.size() >= ending.size()) {
+    return (0 ==
+            str.compare(str.size() - ending.size(), ending.size(), ending));
+  } else {
+    return false;
+  }
+}
+
+Dart_Handle FindDartPluginRegistrantLibrary() {
+  // TODO(): Instead of finding this, it could be passed down from the tool.
```

Comment:
do you plan to work on this issue?

Reply:
Nope.  I have no data to suggest it's worthwhile.

Reply:
How do you envision passing this from the tool?

Reply:
I haven't thought about it.  Maybe we could make it so the library has a unique name so we can look directly for it?

Reply:
ok. I'd suggest filing an issue, and including the general idea about how this should be solved.

Reply:
https://github.com/flutter/flutter/issues/99308